### ISSUE TITLE
Error with inpainting model

### ISSUE THREAD

**Issue by svvenkatakrishnan** [deepinv/deepinv#136]
It appears that the inpainting forward model has an error. 

########Code snippet###########
physics = dinv.physics.Inpainting(
    mask=my_mask,
    tensor_size=x.shape,
    noise_model=dinv.physics.GaussianNoise(sigma=sigma),
    device=device,
)

#Simulate measurements 
y = physics(x)

When I visualize the image y, I notice that there are non-zero values even in the areas where the mask = 0. The noise model should be only applied to the masked portions. 

I realize it may not impact the solver, but I recommend fixing it.

**Comment by matthieutrs** [comment]
Thanks a lot for spotting this mistake! The issue is now solved: the following snippet should run:

```python

# Generate sample   
x = torch.ones((3, 12, 7), device=device).unsqueeze(0)
    
# Generate mask
mask = torch.ones_like(x[0])
mask[0, 0, 0] = 0
mask[1, 1, 1] = 0
mask[2, 2, 2] = 0

# Create physics
physics = dinv.physics.Inpainting(tensor_size=x.shape, mask=mask, device=device)
physics.noise_model = dinv.physics.GaussianNoise(sigma=sigma)
y = physics(x)

# Check that measurements is 0 on the mask, despite noise
assert y[0, 0, 0, 0] == 0
assert y[0, 1, 1, 1] == 0
assert y[0, 2, 2, 2] == 0
```
